spring-boot-persistence/src/main/java/com/baeldung/naming/HibernateConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.baeldung.naming;
+
+import org.hibernate.jpa.boot.spi.IntegratorProvider;
+import org.springframework.boot.autoconfigure.orm.jpa.HibernatePropertiesCustomizer;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class HibernateConfig implements HibernatePropertiesCustomizer {
```

Comment:
I use this class to enable the MetadataExtractorIntegrator. Its goal is to extract the name of the table associated with the entity. I didn't mention in the tutorial because for simplicity I show only the output of the statement but I didn't show the tests. Do you prefer to mention it?